js-core-3/week-1/lesson.md

```diff
@@ -96,6 +130,35 @@ promiseToDoHomeWork
   });
 ```
 
+##### Check-in
+
+**Question (5 mins):**
+
+    Which of the following statements about Promises is false?
+
+    A) A promise starts executing as soon as it is invoked.
```

Comment:
Maybe rephrase this to 'The function passed into a promise is executed as soon as the promise is created.'?